pints/plot.py

```diff
@@ -171,7 +171,12 @@ def function_between_points(f, point_1, point_2, padding=0.25, evaluations=20):
     return fig, axes
 
 
-def histogram(samples, kde=False, ref_parameters=None, n_percentiles=None):
+def histogram(
+        samples,
+        kde=False,
+        ref_parameters=None,
```

Comment:
Maybe put `ref_parameters` at the end? If we're changing stuff anyway :sweat_smile: 

@chonlei @ben18785 @martinjrobins , what do you think the param order here (and in the other functions) should be?

Reply:
What's wrong with `parameter_names` at the end?

Reply:
Just trying to see if there's any particular 'best' order? Like it makes sense to put the two "parameters" arguments next to each other. And then maybe names will be used more often than ref_parameters ?

Reply:
Not helpful but I am completely ambivalent about this! Reckon merge it as, if we can't think of an obvious order, there likely isn't one.

Reply:
:D OK!

In that case I'll just decide. @DavAug would you mind moving `ref_parameters` to the final position, after `parameter_names`, in the two cases we have them? After that I'm happy to merge

Reply:
Yep no problem.